.github/workflows/_build.yml

```diff
@@ -80,7 +80,7 @@ jobs:
         shell: bash
         run: |
           # pid=$(docker run -t -d -w "$WORKDIR" "${GCR_DOCKER_IMAGE}")
-          pid=$(docker run --gpus all --privileged -t -d -w "$WORKDIR" -e "TERM=xterm-256color" "${GCR_DOCKER_IMAGE}")
+          pid=$(docker run --privileged -t -d -w "$WORKDIR" -e "TERM=xterm-256color" "${GCR_DOCKER_IMAGE}")
```

Comment:
I think CI is running on a GPU machine.. at least we run the test on gpu machine

Reply:
The build machine is CPU I think. Only GPU tests are run on GPU machine. I saw this in the log `runner: linux.12xlarge`

PyTorch is built with USE_CUDA=0 in a CUDA container and PyTorch/XLA is built with XLA_CUDA=1. See https://github.com/pytorch/xla/blob/44033edb7b3f4b6a94497ad6083508c5f2dbb838/.circleci/docker/Dockerfile#L15

Reply:
ah sg!